products/solr.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Solr
+permalink: /solr
+alternate_urls:
+  - apache_solr
+releasePolicyLink: https://solr.apache.org/downloads.html#about-versions-and-support
+category: server-app
+changelogTemplate: https://solr.apache.org/docs/{{"__LATEST__" | replace:'.','_'}}/changes/Changes.html
+activeSupportColumn: false
+releaseDateColumn: false
+releaseColumn: true
+sortReleasesBy: 'releaseCycle'
+iconSlug: apachesolr
+releases:
+-   releaseCycle: "9.x"
+    support: true
+    eol: false
+    latest: "9.0.0"
+-   releaseCycle: "8.11.x"
+    support: true
+    eol: false
+    latest: "8.11.2"
```

Comment:
Is it possible to add cycle release date and latest release dates?

Reply:
I added the date for 8.0 but the the 9.0 and 8.11.2 changelog don't have dates:
* https://solr.apache.org/docs/9_0_0/changes/Changes.html
* https://solr.apache.org/docs/8_11_2/changes/Changes.html

Reply:
Used dates from https://solr.apache.org/news.html, https://lists.apache.org/thread/wn9wkmx1ogbf00hj78y2xt3l0bo15ob9

The Docker images seem to lag behind by a few days.